protocol-units/execution/opt-executor/src/executor/initialization.rs

```diff
@@ -70,30 +70,46 @@ impl Executor {
 		(genesis, test_validators)
 	}
 
-	pub fn bootstrap_empty_db(
+	/// Bootstrap a database with a genesis transaction if it is empty.
+	pub fn maybe_bootstrap_empty_db(
 		db_dir: &PathBuf,
 		chain_id: ChainId,
 		public_key: &Ed25519PublicKey,
+		maptos_config: &Config,
```

Comment:
Why you add `maptos_config`, you don't use it in the function.

Reply:
Good catch. I had that in there before I simplified and must have forgotten to remove.

Reply:
Also helps me catch that my Rust Analyzer is not doing much haha.